sushi-bot/index.js

```diff
@@ -299,13 +311,32 @@ module.exports = (clients) => {
 				dailyAsaCounter.max(user, best_score);
 			}
 		}
+
+		{
+			if(text.includes(":exercise-done:")||text.includes(":kintore_houkoku:")){
+				await increment(user, 'exercise-cumulative');
+				Promise.resolve()
+				.then(() => slack.reactions.add({name: 'erai', channel, timestamp}))
+				.then(() => slack.reactions.add({name: 'sugoi', channel, timestamp}))
+	
+				if (channel.startsWith('C')) {
+					unlock(user, 'first-exercise');
+					
+					dailyexerciseCounter.add(user, 1);
+				}
+			}
+		}
 	});
 
 	schedule.scheduleJob('0 19 * * *', async (date) => {
 		dailyAsaCounter.entries().map(([user, score]) => {
 			weeklyAsaCounter.add(user, score);
 		});
 		dailyAsaCounter.clear();
+		dailyexerciseCounter.entries().map(([user, score]) => {
+			exerciseCounter.add(user, score);
```

Comment:
[SHOULD]

```suggestion
			exerciseCounter.add(user, 1);
```

が正しいと思います。